StyleCop.Analyzers/StyleCop.Analyzers/MaintainabilityRules/SA1407SA1408FixAllProvider.cs

```diff
@@ -38,7 +38,7 @@ internal sealed class SA1407SA1408FixAllProvider : DocumentBasedFixAllProvider
                 nodes.Add(node);
             }
 
-            return root.ReplaceNodes(nodes, (originalNode, rewrittenNode) => AddParentheses(originalNode, rewrittenNode));
+            return root.ReplaceNodes(nodes, AddParentheses);
```

Comment:
These changes are actually hurting performance because this will allocate a delegate every time. The lambda expression is cached.